dashboard/modules/job/job_agent.py

```diff
@@ -84,6 +86,34 @@ async def get_job_info(self, req: Request) -> Response:
             content_type="application/json",
         )
 
+    @routes.get("/api/job_agent/jobs/{job_or_submission_id}/logs")
+    @optional_utils.init_ray_and_catch_exceptions()
+    async def get_job_logs(self, req: Request) -> Response:
+        job_or_submission_id = req.match_info["job_or_submission_id"]
+        job = await find_job_by_ids(
```

Comment:
Do we have the following plan to make this call more efficient?

Reply:
we need a new interface `GCS::GetJobInfoBySubmissionID`, for this, we need to make submission_id to a new index of GCS::JobTableData

I will discuss this with @SongGuyang later.